app/js/components/shared/BreadCrumb/BreadCrumb.jsx

```diff
@@ -55,6 +55,9 @@ class BreadCrumb extends Component {
       familyName = patientHeaderDetail.person.personName.familyName;
     }
     let outputBreadcrumb;
+    const contextPath = window.location.href.split('/')[3];
+    const url = localStorage.getItem('returnUrl');
+    const returnUrl = `/${contextPath}/${url}`;
```

Comment:
We may not care if this is not widely used, but generally I've seen code try to make this a bit more resilient to different configurations, specifically I would only add a "/" between the context path and the url if the url does not start with a "/".

Reply:
Thanks @mseaton ! I will remove the potentially extra "/". 

Just FYI, besides the aesthetic aspect of the URL, the extra "/" are harmless, the browser ignores them, the following two URLs produce the same result with no errors or warnings:

- https://humci.pih-emr.org/mirebalais/coreapps/clinicianfacing/patient.page?patientId=106773&app=pih.app.clinicianDashboard

- https://humci.pih-emr.org/mirebalais/////coreapps/clinicianfacing/patient.page?patientId=106773&app=pih.app.clinicianDashboard